app/components/work_packages/date_picker/date_form_component.rb

```diff
@@ -91,14 +90,19 @@ def text_field_options(name:, label:)
         text_field_options
       end
 
-      def caption(name)
+      def today(name:)
         return if duration_field?(name)
 
         text = I18n.t(:label_today).capitalize
 
         return text if @disabled
 
         render(Primer::Beta::Link.new(href: "",
+                                      "aria-label": if name == :start_date
+                                                      I18n.t(:label_today_as_start_date)
+                                                    else
+                                                      I18n.t(:label_today_as_finish_date)
+                                                    end,
```

Comment:
You can construct the string directly with the name parameter. Just keep in mind to change the key to `due_date` in the yml file

```suggestion
                                      "aria-label":  I18n.t(:label_today_as_"#{name.to_s}")
```